src/pages/index.tsx

```diff
@@ -1,10 +1,20 @@
 import type { NextPage } from 'next';
+import { useNews } from 'hooks';
+import { NewsCardType } from 'models/NewsCardType';
+import { NewsCard } from 'components/common/NewsCard/NewsCard';
 
 const Home: NextPage = () => {
+  const { news, isLoading, error } = useNews(process.env.NEXT_PUBLIC_BASE_PATH);
```

Comment:
Please let's move all logic from this component to `components/news/News.tsx` component.

Let's have here only rendering `<News />` component.

So no business logic here.